quantecon/markov/ddp.py

```diff
@@ -401,8 +409,12 @@ def s_wise_max(vals, out=None, out_argmax=None):
         # Check that for every state, at least one action is feasible
         self._check_action_feasibility()
 
-        if not (0 <= beta < 1):
-            raise ValueError('beta must be in [0, 1)')
+        if not (0 <= beta <= 1):
+            raise ValueError('beta must be in [0, 1]')
+        if beta == 1:
+            msg = 'infinite horizon solution methods are disabled with beta=1'
+            warnings.warn(msg)
```

Comment:
This warning is currently being ignored. I suspect this is because we have turned off warnings somewhere else, but thought I'd point it out.